src/components/PokeItem.js

```diff
@@ -1,11 +1,15 @@
 import React, { Component } from 'react'
-
+import { Link } from 'react-router-dom'
 export default class PokeItem extends Component {
     render() {
         return (
          <li>
+            
+             <Link to={`/Pokemon/${this.props.id}`}>
              <img src={this.props.image} alt={this.props.name}></img>
+             </Link>
              {this.props.name}
```

Comment:
this.props.name should be in a html element like `<p>` or `<h2>` and moved to line 9.5